package.json

```diff
@@ -15,7 +15,9 @@
     "socket.io": "^1.7.2"
   },
   "devDependencies": {
+    "css-loader": "^0.26.1",
     "csslint": "^1.0.5",
+    "style-loader": "^0.13.1",
```

Comment:
Looks like `npm install` was missing the `--save-dev` flag when I pulled in these dependencies